google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1123,6 +1125,9 @@ public Builder setHost(String host) {
     /** Enables gRPC-GCP extension with the default settings. */
     public Builder enableGrpcGcpExtension() {
       this.grpcGcpExtensionEnabled = true;
+      if (!userSpecifiedNumChannels) { // If numChannels is not set through options
```

Comment:
We should have comments above the `if` statement instead of on the side?

Reply:
Comments added above if statements